src/classes/CW_CampaignTriggerHandlerTest.cls

```diff
@@ -0,0 +1,266 @@
+@isTest
```

Comment:
Style issue: classes and triggers should use the "UpperCamelCase" notation. Suggestion: rename accordingly.